src/ConvertToSARIF.Engine/DiagnosticRecord.cs

```diff
@@ -17,6 +18,24 @@ public class DiagnosticRecord
 
         public string Message { get; set; }
 
+        public IList<RuleSuppression> Suppression
+        {
+            get
+            {
+                if (suppression == null)
+                {
+                    suppression = new List<RuleSuppression>();
+                }
+
+                return suppression;
+            }
+            set
+            {
+                suppression = value;
+            }
+        }
+        private IList<RuleSuppression> suppression;
```

Comment:
We can also use the `Suppression` property included in the `Microsoft.CodeAnalysis.Sarif`

Reply:
if i set `public IList<Suppression> Suppressions { get;  set; }`
```
if (SuppressionList != null)
{
      foreach (dynamic suppression in SuppressionList)
       {
              Suppression suppressions = new Suppression();
              suppressions.Justification = suppression.Justification;
              suppressions.Kind = SuppressionKind.InSource;

               if (suppressions != null)
               {
                   Suppressions.Add(suppressions);
               }
      }
}
```
when it comes to ` Suppressions.Add(suppressions);`  it will throw an exception:

![image](https://user-images.githubusercontent.com/86460533/127292801-5b67efb5-0fd7-4284-afad-af48ee0698d6.png)

so I used:
```
public IList<Suppression> Suppressions
 {
      get
      {
           if (suppressions == null)
           {
                suppressions = new List<Suppression>();
           }

           return suppressions;
       }
       set
       {
           suppressions = value;
       }
}
private IList<Suppression> suppressions;
```